### PR TITLE
feat: add new details to dkg rituals

### PR DESCRIPTION
**Pull request by evandrosaturnino**
## Description

This PR enhances the DKG rituals table by introducing the following key improvements:

**ID Column Addition:** The id column has been added and the rituals are being sorted by the ID in descending order.
**Pending Transcripts and Aggregations:** Pending transcripts and aggregations are now included, listed accordingly below their respective sections when present.
**Timeout Status:** Now the timeout status can also occur for those awaiting transcripts that exceed the timeout period, providing clarity on their state.

- [x]  [feat: add new details to dkg rituals](https://github.com/threshold-network/tacoscan/commit/020be0d2278c4f105c3b8e0d2e638abd7619f87a)

## Pull Request Type

- [x]  💅 New Feature (Breaking/Non-breaking Change)
- [ ] 🐛 Bug fix (Non-breaking Change: Fixes an issue)
- [ ]  🛠️ Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..

## Notice

- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Closes Issue (If Applicable)

Closes issue #3 
Closes issue #5 
Closes issue #6

## Screenshots (If Applicable)

image

## Testing Please Outline All Testing Steps

1. Clone the repo locally: git clone https://github.com/threshold-network/tacoscan.git
2. Using Node version above 18, the dependecies should be installed with yarn: yarn install
3. Build the graphclient: yarn codegen
4. Build the app: yarn build
5. Preview the app: yarn preview

### CODE REVIEW

File: src/pages/data.jsx [hunk 1]
```diff
@@ -5,6 +5,11 @@ import Web3 from "web3";
 
 const tacoAddr = "0x347cc7ede7e5517bd47d20620b2cf1b406edcf07"
 export const ritual_columns = [
+  {
+    header: "Id",
+    accessor: "id",
+    numeric: false,
```
Comment (cygnusv) [inline src/pages/data.jsx]: ID is an integer

**Review by cygnusv — APPROVED** [review]
Looks good to me! Just one comment regarding ID being an integer

**Comment by manumonti** [comment]
When trying to follow the steps to deploy locally, I get this error when I run yarn build:

```
yarn build
yarn run v1.22.22
$ vite build
vite v4.0.2 building for production...
transforming (18) node_modules/@mui/material/esm/styles/index.jsnode_modules/@mui/styled-engine/index.js (8:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/styled-engine/index.js" was ignored.
transforming (70) node_modules/@mui/system/esm/GlobalStyles/index.jsnode_modules/@mui/system/esm/GlobalStyles/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/GlobalStyles/index.js" was ignored.
node_modules/@mui/system/esm/Box/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Box/index.js" was ignored.
node_modules/@mui/system/esm/createBox.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/createBox.js" was ignored.
node_modules/@mui/system/esm/useThemeProps/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/useThemeProps/index.js" was ignored.
node_modules/@mui/system/esm/ThemeProvider/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/ThemeProvider/index.js" was ignored.
node_modules/@mui/system/esm/useTheme.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/useTheme.js" was ignored.
node_modules/@mui/system/esm/useThemeWithoutDefault.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/useThemeWithoutDefault.js" was ignored.
node_modules/@mui/system/esm/Container/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Container/index.js" was ignored.
node_modules/@mui/system/esm/Unstable_Grid/Grid.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Unstable_Grid/Grid.js" was ignored.
node_modules/@mui/system/esm/Unstable_Grid/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Unstable_Grid/index.js" was ignored.
node_modules/@mui/system/esm/Stack/Stack.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Stack/Stack.js" was ignored.
node_modules/@mui/system/esm/Stack/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Stack/index.js" was ignored.
✓ 228 modules transformed.
Could not resolve "../../.graphclient" from "src/pages/data.jsx"
file: /Users/manumonti/wip/tacoscan/src/pages/data.jsx
error during build:
RollupError: Could not resolve "../../.graphclient" from "src/pages/data.jsx"
    at error (file:///Users/manumonti/wip/tacoscan/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:2287:30)
    at ModuleLoader.handleInvalidResolvedId (file:///Users/manumonti/wip/tacoscan/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:24860:24)
    at file:///Users/manumonti/wip/tacoscan/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:24822:26
error Command failed with exit code 1.
```

I'm running:

```
yarn --version
1.22.22

node --version
v18.20.2
```

File: src/pages/data.jsx (hunk 1 above)
Comment (evandrosaturnino) [reply src/pages/data.jsx]: Addressed in [refactor: reorder ritual table columns](https://github.com/threshold-network/tacoscan/pull/7/commits/c77e9c21b8a418bcf7360cddf6cc507b6652703f)

**Comment by evandrosaturnino** [comment]
> When trying to follow the steps to deploy locally, I get this error when I run yarn build:
> 
> ```
> yarn build
> yarn run v1.22.22
> $ vite build
> vite v4.0.2 building for production...
> transforming (18) node_modules/@mui/material/esm/styles/index.jsnode_modules/@mui/styled-engine/index.js (8:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/styled-engine/index.js" was ignored.
> transforming (70) node_modules/@mui/system/esm/GlobalStyles/index.jsnode_modules/@mui/system/esm/GlobalStyles/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/GlobalStyles/index.js" was ignored.
> node_modules/@mui/system/esm/Box/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Box/index.js" was ignored.
> node_modules/@mui/system/esm/createBox.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/createBox.js" was ignored.
> node_modules/@mui/system/esm/useThemeProps/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/useThemeProps/index.js" was ignored.
> node_modules/@mui/system/esm/ThemeProvider/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/ThemeProvider/index.js" was ignored.
> node_modules/@mui/system/esm/useTheme.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/useTheme.js" was ignored.
> node_modules/@mui/system/esm/useThemeWithoutDefault.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/useThemeWithoutDefault.js" was ignored.
> node_modules/@mui/system/esm/Container/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Container/index.js" was ignored.
> node_modules/@mui/system/esm/Unstable_Grid/Grid.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Unstable_Grid/Grid.js" was ignored.
> node_modules/@mui/system/esm/Unstable_Grid/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Unstable_Grid/index.js" was ignored.
> node_modules/@mui/system/esm/Stack/Stack.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Stack/Stack.js" was ignored.
> node_modules/@mui/system/esm/Stack/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/@mui/system/esm/Stack/index.js" was ignored.
> ✓ 228 modules transformed.
> Could not resolve "../../.graphclient" from "src/pages/data.jsx"
> file: /Users/manumonti/wip/tacoscan/src/pages/data.jsx
> error during build:
> RollupError: Could not resolve "../../.graphclient" from "src/pages/data.jsx"
>     at error (file:///Users/manumonti/wip/tacoscan/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:2287:30)
>     at ModuleLoader.handleInvalidResolvedId (file:///Users/manumonti/wip/tacoscan/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:24860:24)
>     at file:///Users/manumonti/wip/tacoscan/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:24822:26
> error Command failed with exit code 1.
> ```
> 
> I'm running:
> 
> ```
> yarn --version
> 1.22.22
> 
> node --version
> v18.20.2
> ```

@manumonti Please try running the following command `yarn codegen` before the build, please let me know if it worked out.

**Comment by cygnusv** [comment]
This has been working great, so I think we can merge 🚀

**Pull request merged**